front/app/components/SeatInfo/index.tsx

```diff
@@ -43,159 +24,11 @@ const SeatInfo = ({ seatType }: SeatInfoProps) => {
   const hasSeatBasedBillingEnabled = useFeatureFlag({
     name: 'seat_based_billing',
   });
-  const { formatMessage } = useIntl();
-  const { data: appConfiguration } = useAppConfiguration();
-  const { data: seats } = useSeats();
-
-  const maximumSeatNumbers: SeatNumbersType = {
-    admin:
-      appConfiguration?.data.attributes.settings.core.maximum_admins_number,
-    collaborator:
-      appConfiguration?.data.attributes.settings.core.maximum_moderators_number,
-  };
-  const maximumSeatNumber = maximumSeatNumbers[seatType];
-
-  const additionalSeatNumbers: SeatNumbersType = {
-    admin:
-      appConfiguration?.data.attributes.settings.core.additional_admins_number,
-    collaborator:
-      appConfiguration?.data.attributes.settings.core
-        .additional_moderators_number,
-  };
-  const maximumAdditionalSeats = additionalSeatNumbers[seatType];
-
-  // Maximum seat number being null means that there are unlimited seats so we don't show the seat info
-  if (isNil(maximumSeatNumber) || !seats) {
-    return null;
-  }
 
-  let currentSeatNumber = {
-    admin: seats.data.attributes.admins_number,
-    collaborator: seats.data.attributes.project_moderators_number,
-  }[seatType];
-  const additionalSeats = currentSeatNumber - maximumSeatNumber;
-  const hasAdditionalSeats = additionalSeats > 0;
-  const showAdditionalSeats = hasSeatBasedBillingEnabled
-    ? Boolean(hasAdditionalSeats && maximumAdditionalSeats)
-    : hasAdditionalSeats;
-
-  const seatTypeMessage: SeatTypeMessageDescriptor = {
-    admin: messages.currentAdminSeatsTitle,
-    collaborator: messages.currentCollaboratorSeatsTitle,
-  };
-  const currentSeatTypeTitle = seatTypeMessage[seatType];
-  const tooltipMessages: SeatTypeMessageDescriptor = {
-    admin: messages.includedAdminToolTip,
-    collaborator: messages.includedCollaboratorToolTip,
-  };
-  const tooltipMessage = tooltipMessages[seatType];
-
-  const seatTypeInfoMessage = {
-    admin: hasSeatBasedBillingEnabled
-      ? messages.adminInfoTextWithBilling
-      : messages.adminInfoTextWithoutBilling,
-    collaborator: hasSeatBasedBillingEnabled
-      ? messages.collaboratorInfoTextWithBilling
-      : messages.collaboratorInfoTextWithoutBilling,
-  }[seatType];
-
-  const additionalSeatsText = hasSeatBasedBillingEnabled
-    ? `${additionalSeats}/${maximumAdditionalSeats}`
-    : additionalSeats;
-
-  // Show maximum number of seats if user has used more for this value
-  if (currentSeatNumber >= maximumSeatNumber) {
-    currentSeatNumber = maximumSeatNumber;
+  if (hasSeatBasedBillingEnabled) {
```

Comment:
I have separated out the component we display for i1 (Tracker) and that for i2(Billing) to reduce complexity and branching logic. Hopefully this will also help us remove one after 12 has been fully rolled out